cmd/main.go

```diff
@@ -32,6 +33,8 @@ func start() error {
 		Logger: *logger,
 	}
 
+	g := game.New(*app.Logger.Named("game"))
```

Comment:
Not a big fan of single letter names, but it's not worth doing another revision just because of that.

Reply:
fixed